CMPT/Form1.cs

```diff
@@ -549,5 +549,57 @@ private void button1_Click(object sender, EventArgs e)
                 MessageBox.Show(ex.Message);
             }
         }
+
+        private void SearchCustomerButton_Click(object sender, EventArgs e)
+        {
+
+            if (!(string.IsNullOrEmpty(SearchCustomerBox.Text)))
+            {
+                Customer[] customerListLike = database.GetCustomers(SearchByComboBox.SelectedItem.ToString(), SearchCustomerBox.Text);
+
+                CustomersGridView.Rows.Clear();
+
+                foreach (Customer customer in customerListLike)
+                {
+                    CustomersGridView.Rows.Add(customer.AccountNo, customer.LastName, customer.FirstName, customer.StreetNo, customer.StreetName, customer.AptNo, customer.City, customer.PostalCode, customer.PhoneNumber, customer.Email, customer.CreditCard, customer.Rating);
+                }
+            }
+        }
+
+        private void assignActorbox_TextChanged(object sender, EventArgs e)
+        {
+
+        }
+
+        private void makeCopybox_Click(object sender, EventArgs e)
+        {
+            makeCopybox.Text = string.Empty;
+        }
+
+        private void makeCopybutton_Click(object sender, EventArgs e)
+        {
+            int value;
+            try
+            {
+                if(int.TryParse(makeCopybox.Text, out value))
+                {
+                    for (int i = 0; i < value; i++)
+                    {
+                        int rowIdx = movies.CurrentCell.RowIndex;
+
+                        string movieID = movies.Rows[rowIdx].Cells[0].Value.ToString();
+
+                        int lowestCopyID = database.GetLowestAvailableCopyID(movieID);
+
+                        database.MakeCopy(lowestCopyID.ToString(), movieID);
```

Comment:
Adding a copies should update the copy count in table and in the database for the movie